File: JavaScript_Basic_04_2023_SolutionsOfProblems/02.LAB_PB-JS-Conditional-Statements/P07_AreaOfFigures.js
function areaOfFigures(input) {
    let figureType = input[0];
    if (figureType === "square") {
        let a = Number(input[1]);
        console.log((a * a).toFixed(3));
    } else if (figureType === "rectangle") {
        let a = Number(input[1]);
        let b = Number(input[2]);
        console.log((a * b).toFixed(3));
    } else if (figureType === "circle") {
        let r = Number(input[1]);
        console.log((r * r * Math.PI).toFixed(3));
    } else if (figureType === "triangle") {
        let a = Number(input[1]);
        let ha = Number(input[2]);
        console.log((a * ha / 2).toFixed(3));
    }
}
areaOfFigures(["triangle",
    "4.5",
    "20"]);

File: Java_Basic_Jan_2023_SolutionsOfProblems/P02LAB_ConditionalStatements/P07_AreaOfFigures.java
package P02LAB_ConditionalStatements;

import java.util.Scanner;

public class P07_AreaOfFigures {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String figure = scanner.nextLine();
        if (figure.equals("square")) {
            double a = Double.parseDouble(scanner.nextLine());
            System.out.printf("%.3f", a * a);
        } else if (figure.equals("rectangle")) {
            double a = Double.parseDouble(scanner.nextLine());
            double b = Double.parseDouble(scanner.nextLine());
            System.out.printf("%.3f", a * b);
        } else if (figure.equals("circle")) {
            double r = Double.parseDouble(scanner.nextLine());
            System.out.printf("%.3f", Math.PI * r * r);
        } else if (figure.equals("triangle")) {
            double a = Double.parseDouble(scanner.nextLine());
            double h = Double.parseDouble(scanner.nextLine());
            System.out.printf("%.3f", a * h / 2);
        }
    }
}
